### PR TITLE
Update lockup signAndSendTransaction for near-api-js 0.41.0

### PR DESCRIPTION
**Pull request by mehtaphysical**
This fixes the issue caused by updating near-api-js 0.41.0

### CODE REVIEW

**Comment by mehtaphysical** [comment]
This is also only a quick fix. The better approach may be to create a `LockupAccount` class that extends `Account`. Then instead of: https://github.com/near/near-wallet/blob/b9f26062385d69ba0deac928eb3096a93abef0ef/src/utils/wallet.js#L631-L638

Could do:

```js
async getAccount(accountId) {
    return LockupAccount.create(this.connection, accountId)
}
```

**Comment by mehtaphysical** [comment]
This fixes #1730

File: src/utils/account-with-lockup.js [hunk 1]
```diff
@@ -22,7 +22,16 @@ export function decorateWithLockup(account) {
     return decorated;
 }
 
-async function signAndSendTransaction(receiverId, actions) {
+async function signAndSendTransaction(...args) {
+    if(args.length === 1) return signAndSendTransactionV2.bind(this,...args);
```
Comment (vgrichina) [inline src/utils/account-with-lockup.js:26]: this doesn't look right, should be calling function instead of binding it

**Comment by MaximusHaximus** [comment]
@ihaid Could you give this PR a test and in particular verify that working with a 2FA-enabled account works as expected?

**Comment by ihaid** [comment]
Can't disable 2fa here
image

**Comment by esaminu** [comment]
Continuing on https://github.com/near/near-wallet/pull/2248

**Pull request closed without merging**